lib/cocoapods-core/source.rb

```diff
@@ -326,6 +369,18 @@ def specs_dir
       end
     end
 
+    def git_commit_hash
+      (`git rev-parse HEAD` || '').strip
+    end
+
+    def update_git_repo
```

Comment:
we should override this in CocoaPods/CocoaPods to use executable, so the output is still streamed?